azure-pipelines.yml

```diff
@@ -254,3 +254,23 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: RichCodeNav_Indexing
```

Comment:
I believe this has to be moved to a new azure-pipelines.yml because we don't want this rolling up to the main roslyn-CI status checks. Otherwise any failure in this job would fail our CI and block PRs. Having it in a separate .yml also means it can be configured to use master only via the settings at the very top.

Does this need to run on all pull requests or only on a rolling build in master? Does that always cause the indexing job to appear in the CI status for a PR?

Reply:
If we only run it for master CI builds then I think it shouldn't need to show up for PR status. I will change it to a separate yaml file. Could you help set up the pipeline in dnceng/public since I don't have permission to create a new pipeline there?

Reply:
Yeah, we can create the pipeline configuration once it's switched over, that's not a problem.

Reply:
Thanks! I have switched to a separate yaml file. I think it depends on roslyn team's preference so if you like it to run on schedule such as nightly build etc. we could change that too

Reply:
Right now I'd say you tell us how often you want it running; if us running on every master merge is going to overwhelm your systems then we should adjust. :-)

Reply:
Sounds good we should be able to handle that :)